### DISCUSSION TITLE
How can i build the apk instead of aab

### DISCUSSION THREAD

**Discussion by John4650-hub** [Acode-Foundation/Acode#576] [Q&A] [answered]
I have used actions to automate the building process of  Acode but in the log i see that an aab file instead of an apk is generated.

**Comment by deadlyjack** [comment, marked as answer]
you need to specify packageType in `build.json` in root folder of the project

```json
{
  "android": {
    "debug": {
      "packageType": "apk"
    },
    "release": {
      "packageType": "apk"
    }
  }
}
```

**Reply by John4650-hub** [reply]
Thanks.
